autodoc/README.autodoc

```diff
@@ -1,16 +1,15 @@
 Autodoc
-================================================================================
+====================================================================================================
 
-Developer documentation is generated using doxygen
-(https://www.stack.nl/~dimitri/doxygen). If available, this build can use
-graphviz (dot), dia and mscgen to generate graphics like dependency diagrams,
-UML and sequence diagrams.
+Developer documentation is generated using doxygen (https://www.stack.nl/~dimitri/doxygen). If
+available, this build can use graphviz (dot), dia and mscgen to generate graphics like dependency
+diagrams, UML and sequence diagrams.
 
 Documentation is generated nightly and published to the LANL Yellow Network at
-https://rtt.lanl.gov/autodoc.
+https://kellyt.re-pages.lanl.gov/jayenne/draco/index.html.
```

Comment:
This is the first I've heard of re-pages. Is it available to anyone?

Reply:
Yes. This is a part of re-git.